client/src/components/CreateGroup/CreateGroup.js

```diff
@@ -27,11 +29,14 @@ function CreateGroup() {
     let name = form.getFieldValue("name");
     let description = form.getFieldValue("description");
     let avatarURL = form.getFieldValue("logo");
+    let userGroups = users.find((user) => user.user_id === currentUserID).groups;
     if (name === undefined) return;
     let newGroup = {
       name: name,
       description: description,
       avatarURL: avatarURL,
+      user_id: currentUserID,
```

Comment:
Can we not send the list of the groups for a user to the backend? This could cause issues if for some reason the fronted state is out of date. There should be a way to just use mongoose to tell it to add the new groupid to the list of the users groups array.